src/models/PluginToken.js

```diff
@@ -0,0 +1,105 @@
+
+/**
+ * PluginToken model
+ */
+class PluginToken {
+
+    /**
+     * Creates PluginToken model
+     * @param {object} options Options for instance
+     * @param {Array} options.a - Actions
+     * @param {string} options.e - Expiration date (UTC)
+     * @param {number} options.t - Token type (0 - REFRESH, 1 - ACCESS)
+     * @param {string} options.tpc - Permitted topic name
+     */
+    constructor({ a, e, t, tpc } = {}) {
+        this.a = a;
+        this.e = e;
+        this.t = t;
+        this.tpc = tpc;
+    }
```

Comment:
Would it be better to use in the class full named fields, and only in the **toObject()** method convert them to the short version?